packages/core/src/lib/internal/net/StitchAuthDocRequest.ts

```diff
@@ -6,11 +6,13 @@ import { StitchAuthRequest } from "./StitchAuthRequest";
 import { StitchRequest } from "./StitchRequest";
 
 export class StitchAuthDocRequest extends StitchAuthRequest {
-  public readonly document: object;
-
-  public constructor(request: StitchRequest, document: object) {
-    super(request, false);
-    this.document = document;
+  public constructor(
```

Comment:
[minor] you should not be doing anything with `shouldRefreshToken` or `shouldRefreshOnFailure` in `StitchAuthDocRequest`. The superclass should be dealing with this. See the Java and Swift implementations.


The constructor should only accept a `StitchAuthRequest` and a `document`, calling the `StitchAuthRequest` constructor that accepts a `StitchAuthRequest`. I know that javascript doesn't let you have multiple constructors in a class, but there must be a way to do this without breaking the usefulness of inheritance.